src/hdmf/container.py

```diff
@@ -1217,3 +1235,10 @@ def from_dataframe(cls, **kwargs):
         if name is None:
             return cls(data=data)
         return cls(name=name, data=data)
+
+
+class ExternalResourcesManagerContainer(Container, ExternalResourcesManager):
+    def __init__(self, **kwargs):
+        name = 'ExternalResourcesManagerContainer'
+        kwargs['name'] = name
+        super().__init__(**kwargs)
```

Comment:
Does this have a purpose besides testing? If not, then it should live in `tests/unit/common/test_resources.py` or `tests/unit/utils.py`

Reply:
I believe this is also used for the docs.

Reply:
The issue with this type is that it is very narrow in scope and unlikely to be used by others. However, to serve the purpose of a top-level file, we could create a generic `HdmfFile` object and data type in the schema. This is something we have been considering for some time. What do you think about adding a new `HdmfFile` data type to hdmf-common-schema and have it extend the SimpleMultiContainer type? Then we can rename this class here to `HdmfFile`.

Reply:
I think the point was this to be narrow in scope. Until there is a demand for a HdmfFile, I don't think we should make one. The point of this container/manager was initially for testing, but then since a file is required, we needed it for the gallery docs. It's one of those gray areas where the idea is to use an NWBFile, but we don't want to have that in HDMF. Making an HdmfFile seems a bit much for documentation.

Reply:
To round this out, we decided to put this class in both the tests and the documentation examples. If there is a separate need for an `HdmfFile`, we can add it to the schema later on.